### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by sadaf-Daneshgar**
**New feature**
------
- ✅ Add any missing JSX elements to project (without adding extra functionality).
- ✅ This is Bookstore App created with React and CSS.
- ✅ The final design match Zeplin's design (this includes all elements + the same styling)
- ✅ I use the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            </Link>
+          </li>
+          <li className="nav-item">
+            <Link className="nav-link" id="imp3" to="/category">
+              Categories
+            </Link>
+          </li>
+        </ul>
+      </div>
+    </>
```
Comment (ridwanediallo) [inline src/components/Navbar.js]: - [ ] Good working on the naviagation. However, please ensure to add the profile icon as in the template. 

### Your design :point_down: 
Screenshot from 2023-09-28 15-14-28

### The template :point_down: 
Screenshot from 2023-09-28 15-14-49

File: src/components/navbar.css [hunk 2]
```diff
@@ -0,0 +1,334 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: sans-serif;
```
Comment (ridwanediallo) [inline src/components/navbar.css]: - [ ] Please ensure that the font family you pick matches the one in the template. For more information about the font family used, look at the template. You may also take a look at the screenshot below. :point_down: 

### Screenshot 
Screenshot from 2023-09-28 15-04-59

File: src/components/Book.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+          <p className="book-author">{author}</p>
+          <span className="btn" id="commit">
+            Comments
+          </span>
+          <button type="button" className="btn" id="delete" onClick={onDelete}>
+            Remove
+          </button>
+          <span className="btn" id="edit">
+            Edit
+          </span>
```
Comment (ridwanediallo) [inline src/components/Book.js]: - [ ] Please to match the original design ensure to add to dividers as I pointed it out on the screenshot on top of this comment.  👌😊

File: src/components/Book.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+          <h5>{isCompleted ? 'completed' : 'not completed'}</h5>
+        </div>
+        <div className="current-part">
+          <p className="current-chapter">CURRENT CHAPTER</p>
+          <h4>Chapter</h4>
+          <button type="button" className="upd-btn" onClick={toggleStatus}>
+            {isCompleted ? 'Mark as Incomplete' : 'Mark as Complete'}
+          </button>
+        </div>
+      </div>
```
Comment (ridwanediallo) [inline src/components/Book.js]: - [ ] Please to match the original design I would suggest you add the divider as I pointed out in the screenshot on top of this comment.  👌😊

File: src/components/BookForm.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+            <option value="Category">Category</option>
+            <option value="Fiction">Fiction</option>
+            <option value="General">General</option>
+            <option value="Amazing">Amazing</option>
+          </select>
+          <button type="button" className="addbtn" onClick={handleAddBtn}>
+            ADD BOOK
+          </button>
+        </div>
+      </div>
```
Comment (ridwanediallo) [inline src/components/BookForm.js]: - [ ] Please ensure that the font family you pick matches the one in the template. For more information about the font family used, look at the template. You may also take a look at the screenshot below.  :point_down: 

### Screenshot :point_down: 
Screenshot from 2023-09-28 15-41-29

**Review by ridwanediallo — APPROVED** [review]
# Changes Required ♻️
## Hi @sadaf-Daneshgar , 
AlmostThereReachGIF

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Your general layout looks exactly like the template. ✔


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookForm.js (hunk 5 above)
Comment (sadaf-Daneshgar) [reply src/components/BookForm.js]: Hello @ridwanediallo thank you for your nice review I added this font-families it take error that is why I deleted.

Comment (ridwanediallo) [reply src/components/BookForm.js]: @sadaf-Daneshgar can you reach out on zoom to see if I can help with that?

Comment (sadaf-Daneshgar) [reply src/components/BookForm.js]: @ridwanediallo  if I add just like this `f font-family: 'Montserrat';` it take the linters error but if I add like this ` font-family: 'Montserrat', sans-serif;` it work is the second way that I used is correct??

**Review by ademibowale — APPROVED** [review]
Hi @sadaf-Daneshgar  


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️
- Styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/)✔️
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good use of Github flow ✔️






- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @Adebowale ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**